lib/js/components/SurveyQuestions/SurveyQuestionScale/SurveyQuestionScale.vue

```diff
@@ -154,22 +194,52 @@
 	&__elaborationInput {
 		margin-top: $space-xxs;
 	}
+
+	&__container {
+		display: flex;
+		flex: 1;
+		flex-direction: row;
+		gap: $space-l;
+
+		&.-justifyEnd {
+			justify-content: flex-end;
+		}
+
+		&.-oneContainer {
+			justify-content: space-between;
+		}
+
+		&.-hideOnMobile {
+			display: none;
+
+			@media #{breakpoint-s()} {
+				display: flex;
+			}
+		}
+	}
+}
+
+.-visibleOnMobile {
```

Comment:
We already have `-hideOnMobile` so I'd rename it to `-hideOnDesktop`. Hiding is what it actually does.